update-version-wrapper

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+export GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa.gpii-ci'
+git config --global user.email "gpii-bot@gpii.net"
+git config --global user.name "GPII Bot"
+# Don't need the full history, but want a few previous revisions in case we
+# need those revisions to push to a remote (e.g. gitlab) that hasn't been
+# updated in a bit.
+git clone --depth 10 git@github.com:gpii-ops/gpii-terraform
+cd gpii-terraform/modules/deploy
+git remote add gitlab git@gitlab.com:gpii-ops/gpii-terraform
+
+while true ; do
+    echo "Starting version check at $(date)"
+    git fetch --all --tags -p
+    git reset --hard origin/master
+
+    ./update-version
+    if [ -n "$(git status --porcelain -- version.yml)" ]; then
+        git add version.yml
+        git commit -m"[auto] Update version.yml"
+        git push origin HEAD
+        git push gitlab HEAD
+    fi
+
+    sleep_secs=300
+    echo "Sleeping $sleep_secs..."
+    sleep $sleep_secs
+done
```

Comment:
shellcheck warnings:

```
Line 10:
cd gpii-terraform/modules/deploy
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```

Reply:
Hmm, my shellcheck (version 0.3.8) doesn't report this one. I switched to `set -e` to guard against unrecoverable errors.